docs/events/glides/index.mdx

```diff
@@ -0,0 +1,165 @@
+They could differ if sending the input was delayed due to network problems, or if the device's clock is wrong.
+
+Consumers are recommended to use the server-reported event time for most use cases, and not this client-reported `inputTimestamp` field, because:
+
+- The `inputTimestamp` is generated by the client device, and so is in some sense less trustworthy. What if a client's clock is set wrong? We don't do any validation, and we can't, because we can't tell the difference between an incorrect time and a correct time that was delayed by network issues. This isn't so much a specific concern as a general practice of not depending on non-validated client data.
+- Glides applies updates in the order they're received. If a consumer applies events based on client-provided timestamps, they could apply events in a different order than Glides and end up with incorrect results about the state of a trip.
+- The CloudEvents docs seem to say that the event time's consistency is more important than it being the original occurence time. Server-provided times will be more consistent than client-provided times.
+
+CloudEvents [`suggests`](https://github.com/cloudevents/spec/blob/v1.0.2/cloudevents/spec.md#time) that the event time should be "when the occurence happened", which would be when the input was made,
+but for the reasons above, Glides uses a server-reported time as the event time and separates the client-reported `inputTime` into `metadata`.
+
+### Operator
+
+Fields:
+
+- `badgeNumber` (string)
+
+MAY refer to other non-operator Area 132 employees in some cases, for example if an inspector signs in as fit for duty in Glides (they aren't required to).
+
+It is represented as an object to provide future extensibility if needed.
+
+## Details
+
+Events are in the [CloudEvents](https://github.com/cloudevents/spec/blob/v1.0.2/cloudevents/spec.md) format, and published via AWS Kinesis. See [RFC 19 Event Driven Architecture](https://github.com/mbta/technology-docs/blob/main/rfcs/accepted/0019-event-driven-architecture.md) for more about CTD's approach to publishing events.
+
+### Kinesis
+
+Events will be written as records to a Kinesis stream. Each Glides environment (`dev`, `dev-green`, and `prod`) will have a separate stream, named `ctd-glides-<environment>`.
+
+The partition key will be a hash of the station at which the inspector is working and the inspector's identity, which ensures that multiple events from a single inspector are ordered correctly if the records are distributed across multiple shards.
+
+Multiple events can be included in a single Kinesis record, by wrapping them in a JSON array. This is an optimization for improving write speeds: further steps in the Kinesis pipeline may break up or rearrange these arrays.
+
+Kinesis only adds a limited (<1s) latency between when the event is generated and when it is available to consumers.
```

Comment:
suggestion: It looks like this is messing up the Github syntax highlighting: 
![Screenshot 2023-07-07 at 8 44 14 AM](https://github.com/mbta/schemas/assets/214921/214c8801-c0d1-437f-bb41-72ffdde025bf)
```suggestion
Kinesis only adds a limited (&lt;1s) latency between when the event is generated and when it is available to consumers.
```